clustered/common/src/main/java/org/ehcache/clustered/common/internal/messages/EhcacheCodec.java

```diff
@@ -19,7 +19,6 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.terracotta.entity.MessageCodec;
-import org.terracotta.entity.MessageCodecException;
```

Comment:
I know they're not needed... but could you reinstate this (and the LockSyncMessaging) imports to keep these files out of the diff.

Reply:
fixed